src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -588,6 +664,8 @@ def _update_ralph(self):
         for project_id in self.openstack_projects:
             self._add_project(self.openstack_projects[project_id], project_id)
 
+        self._match_physical_and_cloud_hosts()
```

Comment:
could you run this only if some argument is passed to this command (ex. `ralph openstack_sync --match-ironic-physical-hosts`)?